addons/api/addon/generated/models/host-catalog.js

```diff
@@ -104,12 +109,14 @@ export default class GeneratedHostCatalogModel extends BaseModel {
   role_external_id;
```

Comment:
do `role_arn` and `role_external_id` also need `for: 'plugin'` option or is there a reason to not add it?

Reply:
ah good catch, I forgot to add `plugin` for those two fields. wiIl update it